test/integration/bastion/bastion_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package bastion_test
```

Comment:
license header missing

Reply:
fixed in https://github.com/gardener/gardener-extension-provider-gcp/pull/288/commits/33570f2898da5b814bb3aebcb5c8766c0f4be77a